openshift/ftest/src/test/java/org/arquillian/cube/openshift/ftest/HelloPodDeploymentOpenShiftTestCase.java

```diff
@@ -9,11 +9,13 @@
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Assert;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 @RunWith(ArquillianConditionalRunner.class)
 @RequiresOpenshift
+@Ignore
 public class HelloPodDeploymentOpenShiftTestCase {
```

Comment:
We can put the explanation to `@Ignore` annotation.

The only thing I'm concerned about is - if we ignore it - it's equal to just deleting it in this case. Can't we fix it so we can run in the CI? Or maybe write another test which is less resource consuming?

Reply:
This is the only one test in which we are not using standalone. We are using `arquillian-junit-container` where we are building docker image as a part of tests from test resources & then deploy it. Let me try to use already built image & run this tests.